lib/node_modules/@stdlib/blas/base/scasum/src/addon.c

```diff
@@ -41,4 +41,21 @@ static napi_value addon( napi_env env, napi_callback_info info ) {
 	return out;
 }
 
-STDLIB_NAPI_MODULE_EXPORT_FCN( addon )
+/**
+* Receives JavaScript callback invocation data.
+*
+* @param env    environment under which the function is invoked
+* @param info   callback data
+* @return       Node-API value
+*/
+static napi_value addon_method( napi_env env, napi_callback_info info ) {
+	STDLIB_NAPI_ARGV( env, info, argv, argc, 4 );
+	STDLIB_NAPI_ARGV_INT64( env, N, argv, 0 );
+	STDLIB_NAPI_ARGV_INT64( env, strideX, argv, 2 );
+	STDLIB_NAPI_ARGV_INT64( env, offsetX, argv, 3 );
+	STDLIB_NAPI_ARGV_STRIDED_COMPLEX64ARRAY( env, CX, N, strideX, argv, 1 );
+	STDLIB_NAPI_CREATE_DOUBLE( env, (double)API_SUFFIX(c_scasum_ndarray)( N, CX, strideX, offsetX ), out );
```

Comment:
```suggestion
	STDLIB_NAPI_ARGV_STRIDED_COMPLEX64ARRAY( env, CX, N, strideX, argv, 1 );
	STDLIB_NAPI_CREATE_DOUBLE( env, (double)API_SUFFIX(c_scasum_ndarray)( N, (void *)CX, strideX, offsetX ), out );
```

This change also applies to L40.